README.md

```diff
@@ -60,20 +63,27 @@ Once you have installed these prerequisites, you can proceed with the customizat
 
 
 #### 1. Install the extensions
-To install the extensions, run the following command: 
-##### on Mac OS or linux:
+To install the extensions, run the following command:
+##### on Mac OS:
 ```bash
 > cat extensions.txt | xargs -L 1 code --install-extension
 ```
-
-This command installs each extension listed in the `extensions.txt` file using the `code --install-extension` command.
+##### on Linux:
+```bash
+> cat extensions.txt | xargs -L 1 codium --install-extension
```

Comment:
Is it valid for every Linux distro, or is it just for Arch? I remember using the `code` command on Ubuntu, for example, but that was a long time ago

Reply:
> Is it valid for every Linux distro, or is it just for Arch? I remember using the `code` command on Ubuntu, for example, but that was a long time ago

You are right, I am using the de-M$'ed MIT license version that does not call home, that should be "Code" for most installs.   

So far, the needs are 
- OS detection/variegation 
- Version of Code used (some will be using the MIT license version)

Reply:
Also, there's this https://github.com/drcika/apc-extension?tab=readme-ov-file#mac-and-linux-users which explains why my issue occurs, and how to correct. I suggest possibly, two versions, e.g. settings_MACOS.json or settings_LINUX.json

Reply:
OK I have it working as you have shown, I have been required to make more than minor but not quite major edits for correct functioning of the APC plugin, as well as file path corrections for the variance between OSes.  Edits to the README.md are also needed, as the APC issue seems to be common enough to include, as it's a permissions issue.